app/models/concerns/user_concern.rb

```diff
@@ -136,6 +136,13 @@ def using_acls?
       self[:permission_context].to_s == 'acls'
     end
 
+    def self.available_permission_contexts
+      {
+        acls: 'Access Controls (modern granular access)',
+        role_based: 'Legacy Access Controls (current)',
```

Comment:
what does "current" mean here? 🤔  I think "Legacy Access Controls" would probably be sufficient..

Reply:
Yeah, I just wanted to make it really clear, that's what 99.9% of the people should use, but I'll remove.  Actually, I think I'll change it to "Role-Based Access".  I hesitate to call it legacy just yet.